### DISCUSSION TITLE
[Feature] Offline Support

### DISCUSSION THREAD

**Discussion by cinderblockgames** [immich-app/immich#14417] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'm on Android 1.121.0 build.168.  When I open the app when I'm not on my network, it doesn't load any pictures I've taken since the last time I was on my network.  It would be nice to be able to manage local files still.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4466
